lib/datura/to_es/pdf_to_es/fields.rb

```diff
@@ -0,0 +1,325 @@
+  def date_display
+    # Datura::Helpers.date_display(date)
+  end
+
+  def date_not_after
+    # if @row["date_not_after"] && !@row["date_not_after"].empty?
+    #   Datura::Helpers.date_standardize(@row["date_not_after"], false)
+    # else
+    #   date(false)
+    # end
+  end
+
+  def date_not_before
+    # if @row["date_not_before"] && !@row["date_not_before"].empty?
+    #   Datura::Helpers.date_standardize(@row["date_not_before"], true)
+    # else
+    #   date(true)
+    # end
+  end
+
+  def description
+    # @row["description"]
+  end
+
+  def extent
+    # @row["extent"]
+  end
+
+  def format
+    # @row["format"]
+  end
+
+  def image_id
+    # @row["image_id"]
+  end
+
+  def keywords
+    # if @row["keywords"]
+    #   @row["keywords"].split("; ")
+    # end
+  end
+
+  def language
+    # @row["language"]
+  end
+
+  def languages
+    # if @row["languages"]
+    #   @row["languages"].split("; ")
+    # end
+  end
+
+  def medium
+    # @row["medium"]
+  end
+
+  # nested field
+  def person
+    # if @row["person"]
+    #   @row["person"].split("; ").map do |p|
+    #     { "name" => p }
+    #   end
+    # end
+  end
+
+  def places
+    # if @row["places"]
+    #   @row["places"].split("; ")
+    # end
+  end
+
+  def publisher
+    # @row["publisher"]
+  end
+
+  # nested field
+  def recipient
+    # if @row["recipient"]
+    #   @row["recipient"].split("; ").map do |p|
+    #     { "name" => p }
+    #   end
+    # end
+  end
+
+  def relation
+    # @row["relation"]
+  end
+
+  def rights
+    # @row["rights"]
+  end
+
+  def rights_holder
+    # @row["rights_holder"]
+  end
+
+  def rights_uri
+    # @row["rights_uri"]
+  end
+
+  def source
+    # @row["source"]
+  end
+
+  # nested field
+  def spatial
+  end
+
+  def subjects
+
+  end
+
+  def subcategory
+    @row["subcategory"]
+  end
+
+  # text is generally going to be pulled from
+  def text
+    text_all = []
+    @pdf.pages.each do |page|
+      text_all << page.text
+    end
+    text_all += text_additional
+    text_all = text_all.compact
+    Datura::Helpers.normalize_space(text_all.join(" "))[0..@options["text_limit"]]
+  end
+
+  # override and add by collection as needed
+  def text_additional
+    [ title ]
+  end
+
+  def title
+    @filename.delete_suffix(".pdf")
+  end
+
+  def title_sort
+    Datura::Helpers.normalize_name(title) if title
+  end
+
+  def topics
+    # if @row["topics"]
+    #   @row["topics"].split("; ")
+    # end
+  end
+
+  def type
+    # @row["type"]
+  end
+
+  def uri
+    File.join(
+      @options["site_url"],
+      "item",
+      @id
+    )
+  end
+
+  def uri_data
+    File.join(
+      @options["data_base"],
+      "data",
+      @options["collection"],
+      "source/pdf",
+      "#{@filename}.pdf"
+    )
+  end
+
+  def uri_html
+    File.join(
+      @options["data_base"],
+      "data",
+      @options["collection"],
+      "output",
+      @options["environment"],
+      "html",
+      "#{@id}.html"
+    )
+  end
```

Comment:
We want this to be blank related to https://github.com/whitmanarchive/whitman-issues/issues/598. This `def` is repeated below where it is emptied, but this definition should be removed to avoid confusion in the future.